.github/CODEOWNERS

```diff
@@ -904,6 +904,7 @@
 /types/ftdomdelegate/                                    @dotnetnerd
 /types/ftp/                                              @rogierschouten
 /types/ftpd/                                             @rogierschouten
+/types/fuelux                                            @xiaoLiu
```

Comment:
Don't edit this manually, we have a script to update it.

Reply:
ok